chingsley/ireporter
# Ft postredflag

## Description:
This pull request deals with the implementation of the route to create a new redflag (POST/redflags)
## How to test this route
in package.json, make sure the script contains the test attribute:
"test": "export NODE_ENV=test && nyc --reporter=lcov mocha --recursive ./tests/* --require babel-register --exit"
Then run the command: npm test.
Also the route can be tested using postman. Enter the address 'http://localhost[port number]/api/v1/redflags'. Send the req with the necessary payload : email, location, type, comment, (optionally: images, videos). The hit send.
